### ISSUE TITLE
Network Graph, Basic Tooltip

### ISSUE THREAD

**Issue by adonahue** [rifflearning/zenhub-platform-project#102]
As a user, it's more useful to me to have more information about who I connect with, so that I understand how I'm interacting with my peers. 

** Story Acceptance Criteria **
The goal of this story is to add basic tool tip functionality that displays user information for each node. 
* A design for this story is here, Wireframe: "Network graph 2" Version: "Basic Tooltip"
https://drive.google.com/open?id=1uSoqdZEDtI0DR_urOweDE34U17tocSRW
- [x] The tooltip of each person node should display their EDU username and contact count. 
- [x] Each cluster node should display the cluster name and contact count.

**Comment by adonahue** [comment]
From talking with @brecriffs - I believe this has been implemented on PRs for other stories in this feature. Moving to QA so it can be tested once that code is merged and deployed.

**Comment by ebporter** [comment]
This can't be moved into accepted until #157 is merged. Is that correct @brecriffs?

**Comment by brecriffs** [comment]
@ebporter That's correct.

**Comment by adonahue** [comment]
Functionality / code for this story is in the PR(s) attached to story #157.

**Comment by adonahue** [comment]
Accepted as complete on staging. Some variation from initial design but meets requirements. Will create a new story to address a11y criteria.
